Tests/BasicTestApp.Server/Program.cs

```diff
@@ -1,22 +1,21 @@
-﻿using Microsoft.AspNetCore;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-
-namespace BasicTestApp.Server;
-
-public class Program
-{
-    public static void Main( string[] args )
-    {
-        BuildWebHost( args ).Run();
-    }
-
-    public static IWebHost BuildWebHost( string[] args ) =>
-        WebHost.CreateDefaultBuilder( args )
-            .UseConfiguration( new ConfigurationBuilder()
-                .AddCommandLine( args )
-                .Build() )
-            .UseStartup<Startup>()
-            .UseStaticWebAssets()
-            .Build();
-}
\ No newline at end of file
+﻿using Microsoft.AspNetCore.Builder;
```

Comment:
Prefer to not use top-level statements.

Reply:
I'll refactor then. I think I copied from somewhere since I was having trouble with the WebApplicationFactory not resolving how it should build the application successfully with the old version.